src/mixed.js

```diff
@@ -92,6 +89,9 @@ const proto = (SchemaType.prototype = {
   __isYupSchema__: true,
 
   constructor: SchemaType,
+  
+  _whitelist: new RefSet(),
+  _blacklist: new RefSet(),
```

Comment:
this moves the properties to the prototype which is incorrect. They are instance properties and should not be shared between all instances of SchemaType, which is what would happen here.